editoast/fga/src/client.rs

```diff
@@ -42,9 +43,10 @@ pub const DEFAULT_OPENFGA_MAX_CHECKS_PER_BATCH_CHECK: u32 = 50;
 pub const DEFAULT_OPENFGA_MAX_TUPLES_PER_WRITE: u64 = 100;
 
 #[derive(Debug, Clone)]
+#[cfg_attr(any(test, feature = "test_utilities"), derive(Default))]
```

Comment:
Not sure it makes sense to derive Default here, even in tests. `impl Default` with sensible values would maybe be better, but when the default isn't straightforward, I think a test constructor is better (`Store::new_test`), wdyt?

Reply:
not needed anymore as discussed